functions/pyenv-init.fish

```diff
@@ -0,0 +1,16 @@
+
+function pyenv-init --description "Pyenv initialization script for fish shell"
+
+    # Checks if directories already exists. If not, creates then
```

Comment:
It could also be solved by the *init* function. It is not necessary to run that on every new session.